sql/derived_tables/po_lines_locations.sql

```diff
@@ -1,40 +1,41 @@
-DROP TABLE IF EXISTS po_lines_locations;
+/* This query creates a local table for purchase order line locations including location quantity and type.
+It uses conditional statements to search for location ID and location name in the 
+purchase order line location and holdings location fields, displays whichever is not null,
+and indicates the source of the resulting location data (purchase order location or purchase
+order holding location) in the pol_location_source column. 
+*/
 
--- Create a local table for Purchase Order Line Locations including location quantity and type.
-CREATE TABLE po_lines_locations AS
-WITH pol_location AS (
-    SELECT
-        pol.id AS pol_id,
-        json_extract_path_text(locations.data, 'locationId') AS pol_loc_id,
-        json_extract_path_text(locations.data, 'quantity') AS pol_loc_qty,
-        json_extract_path_text(locations.data, 'quantityElectronic') AS pol_loc_qty_elec,
-        json_extract_path_text(locations.data, 'quantityPhysical') AS pol_loc_quant_phys
+DROP TABLE IF EXISTS folio_reporting.po_lines_locations;
+CREATE TABLE folio_reporting.po_lines_locations AS
+
+SELECT
+	pol.id AS pol_id,
+	json_extract_path_text(locations.data, 'quantity') AS pol_location_qty,
+	json_extract_path_text(locations.data, 'quantityElectronic') AS pol_loc_qty_elec,
+	json_extract_path_text(locations.data, 'quantityPhysical') AS pol_loc_quant_phys,
```

Comment:
I think this column name should be `pol_loc_qty_phys` as it is called in the create index statement below.